packages/unit-testing/stubs/crypto/crypto.js

```diff
@@ -0,0 +1,145 @@
+define(['./SecretKey', './Hash', './Hmac', './Cipher', './Decipher'], function (SecretKey, Hash, Hmac, Cipher, Decipher) {
+    /**
+     * SuiteScript crypto module
+     * The N/crypto module encapsulates hashing, hash-based message authentication (hmac), and symmetrical encryption.
+     *
+     * @module N/crypto
+     * @suiteScriptVersion 2.x
+     *
+     */
+    var crypto = function () { };
+
+    /**
+     * Method used to create a new crypto.SecretKey object.
+     * @param {Object} options The options object.
+     * @param {string} options.guid A GUID used to generate a secret key.
+     * @param {string} options.secret Secret for the key
+     * @param {string} options.encoding Specifies the encoding for the SecureKey.
+     * @return {SecretKey}
+     * @throws {SuiteScriptError} MISSING_REQD_ARGUMENT If options or mandatory parameter are undefined.
+     * @since 2015.1
+     */
+    crypto.prototype.createSecretKey = function (options) { };
+
+    /**
+     * Method used to create a crypto.Hash object.
+     * @param {Object} options
+     * @param {string} options.algorithm The hash algorithm. Set using the crypto.HashAlg enum.
+     * @return {Hash}
+     * @throws {SuiteScriptError} MISSING_REQD_ARGUMENT If options or mandatory parameter are undefined.
+     * @since 2015.1
+     */
+    crypto.prototype.createHash = function (options) { };
+
+    /**
+     * Method used to create a crypto.Hmac object.
+     * @param {Object} options The options object.
+     * @param {string} options.algorithm The hash algorithm. Use the crypto.HashAlg enum to set this value.
+     * @param {SecretKey} options.key The crypto.SecretKey object.
+     * @return {Hmac}
+     * @throws {SuiteScriptError} MISSING_REQD_ARGUMENT If options or mandatory parameter are undefined.
+     * @since 2015.1
+     */
+    crypto.prototype.createHmac = function (options) { };
+
+    /**
+     * Method used to create and return a crypto.EncryptionAlg object.
+     * @governance none
+     * @param {object} options The options object.
+     * @param {string} options.algorithm The hash algorithm. Set the value using the crypto.EncryptionAlg enum.
+     * @param {SecretKey} options.key The crypto.SecretKey object.
+     * @param {string} options.blockCipherMode
+     * @param {string} [options.padding] The padding for the cipher text.
+     * @return {Cipher}
+     * @throws {SuiteScriptError} MISSING_REQD_ARGUMENT If options or mandatory parameter are undefined.
+     * @since 2015.1
+     */
+    crypto.prototype.createCipher = function (options) { };
+
+    /**
+     * Method used to create a crypto.Decipher object.
+     * @param {object} options The options object.
+     * @param {string} options.algorithm The hash algorithm. Set by the crypto.EncryptionAlg enum.
+     * @param {string} options.key The crypto.SecretKey object used for encryption.
+     * @param {string} options.blockCipherMode
```

Comment:
blockCipherMode doesn't appear in help
https://nlcorp.app.netsuite.com/app/help/helpcenter.nl?fid=section_4358650886.html